OpenAI-DotNet/Models/Model.cs

```diff
@@ -21,7 +21,7 @@ public Model(string id)
         /// Allows a model to be implicitly cast to the string of its id.
         /// </summary>
         /// <param name="model">The <see cref="Model"/> to cast to a string.</param>
-        public static implicit operator string(Model model) => model.Id;
+        public static implicit operator string(Model model) => model?.Id;
```

Comment:
This probably shouldn't be nullable. If anything we should default to the API default engine.

Could you give me your complete error stack?